dashboard/lib/routes/common.js

```diff
@@ -52,7 +53,37 @@ function parseRoles(roles) {
 
     return '';
 }
+/**
+ * Convert to lower case and cut the last character
+ * @param {string} regionName
+ * @returns {string} ready for compare
+ */
+function getRegionNameForCompare(regionName) {
+
+    return (regionName.substring(0, regionName.length - 1)).toLowerCase();
 
+}
+/**
+ * Check if region is authorized for an username using list in config file
+ * @param {Object} region
+ * @param {string} regionName
+ * @param {string} username
+ */
+function checkAuthorizedByConfig(region, regionName, username) {
+    if (!region.authorized) {
+
+        var data = config.idm.regionsAuthorized;
+        data.filter(function (obj) {
+            var key = Object.keys(obj).toString();
+            var configRegionName = getRegionNameForCompare(key);
```

Comment:
When checking authorization by config, why do you need to get a substring of the username with the region name if you will have the whole username in config?